bigtable/bigtable.go

```diff
@@ -49,7 +49,7 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-const prodAddr = "bigtable.googleapis.com:443"
+const prodAddr = "bigtable.UNIVERSE_DOMAIN:443"
```

Comment:
how does this work? I assume `WithDefaultUniverseDomain` treats this as a template. (Is WithDefaultUniverseDomain part of gax?)

Maybe a comment here would be helpful